README.md

```diff
@@ -237,53 +254,29 @@ $ cd modality-probe-cli
 $ cargo install --path .
 $ cd ../
 $ modality-probe manifest-gen \
-    --lang rust \
-    --file-extension rs \
+    --lang c \
+    --file-extension c \
     --component-name example-component \
     --output-path example-component \
-    examples/rust-example
+    examples/c-example
 ```
 
 Next, we'll want to generate the source code that gives those symbols
 we discussed in the code snippet examples in the previous section
 their definitions. To do that, we'll use `header-gen`:
 
 ```shell
+$ mkdir -p /examples/c-example/include
 $ modality-probe header-gen \
-    --lang rust \
-    --output-path examples/rust-example/src/component_definitions.rs \
+    --lang c \
+    --output-path ../examples/c-example/include/component_definitions.h \
```

Comment:
Can axe the leading `../` on the path since it seems like we're running these commands from the root.

Reply:
:+1: